test/layer.spec.ts

```diff
@@ -113,7 +113,21 @@ describe("applyLayers", () => {
       code: lambda.Code.fromInline("test"),
       handler: "hello.handler",
     });
-    const errors = applyLayers(stack, stack.region, [hello1, hello2], PYTHON_LAYER_VERSION, NODE_LAYER_VERSION);
+
+    const hello3 = new lambda.Function(stack, "HelloHandler3", {
+      runtime: lambda.Runtime.NODEJS_14_X,
+      code: lambda.Code.fromAsset("test/lambda"),
+      handler: "example-lambda.handler",
+    });
+
+    const errors = applyLayers(
+      stack,
+      stack.region,
+      [hello1, hello2, hello3],
+      PYTHON_LAYER_VERSION,
+      NODE_LAYER_VERSION,
+      NODE_LAYER_VERSION,
```

Comment:
I think the 2 `NODE_LAYER_VERSION` parameters may be a mistake, also I know the `PYTHON_LAYER_VERSION` was already there but since all 3 of the lambdas for this unit test are Nodejs I do not think we need to pass in a `PYTHON_LAYER_VERSION`

Reply:
Ah yeah, you're correct! That last parameter was for the extension layer version, I mistakenly assumed it was one version per item in the lambda array - fixed in latest commit